### PR TITLE
155847048 operation area drawing

### PR DESCRIPTION
**Pull request by mmarkuss**
# Fixed
- Hide operation drawing tools by default.
- Let user select drawing tools for primary and secondary operation areas.

### CODE REVIEW

File: ote/src/cljs/ote/views/place_search.cljs [hunk 1]
```diff
@@ -186,8 +194,30 @@
                          :on-update-input #(e! (ps/->SetSecondaryPlaceName %))
                          :search-text (or (:secondary-name place-search) "")
                          :on-new-request #(e! (ps/->AddPlace (aget % "id") false))}]]
+
      [:div {:style {:width "70%"}}
-       [places-map e! (:results place-search)]]]))
+       [places-map e! (:results place-search) (:show? place-search)]
+
+      [:span
```
Comment (rckopteri) [inline ote/src/cljs/ote/views/place_search.cljs]: I prefer divs

File: ote/src/cljs/ote/app/controller/place_search.cljs [hunk 2]
```diff
@@ -66,6 +67,13 @@
   (process-event [{name :name} app]
     (search (assoc-in app [:place-search :secondary-name] name) false))
 
+  SetDrawControl
+  (process-event [{show? :show? primary? :primary?} app]
+    (if (nil? primary?)
+      (assoc-in (update-in app [:place-search] dissoc :primary?) [:place-search :show?] show?)
```
Comment (rckopteri) [inline ote/src/cljs/ote/app/controller/place_search.cljs]: I don't know if this is ok, but I would suggest to change this like this:
(-> app
 ( update-in ...)
(assoc-in ....)
(assoc-in ...)
(assoc-in ...)

File: ote/src/cljs/ote/views/place_search.cljs (hunk 1 above)
Comment (mmarkuss) [reply ote/src/cljs/ote/views/place_search.cljs]: I'll leave this span which Toni recommended using here.

File: ote/src/cljs/ote/app/controller/place_search.cljs (hunk 2 above)
Comment (mmarkuss) [reply ote/src/cljs/ote/app/controller/place_search.cljs]: Fixed.

**Pull request merged**